lib/cartopy/tests/mpl/test_contour.py

```diff
@@ -61,3 +61,12 @@ def test_contour_linear_ring():
     ax.contourf(lons, lats, vals, np.arange(9), transform=ccrs.PlateCarree())
 
     plt.draw()
+
+
+def test_contour_update_bounds():
+    """Test that contour updates the extent"""
+    xs, ys = np.meshgrid(np.linspace(0, 360), np.linspace(-80, 80))
+    zs = ys**2
+    ax = plt.axes(projection=ccrs.Orthographic())
+    ax.contour(xs, ys, zs, transform=ccrs.PlateCarree())
+    assert len(ax.get_extent()) == 4
```

Comment:
Isn't this always true? I'm not sure what is being asserted here.

Reply:
This was probably a misleading way to test it, that failed because `get_extent()` would raise and not return the bounds. I've updated it to just be a smoke-test drawing and making sure it doesn't fail.

I suppose this brings up a different issue that `ax.get_extent()` should really never fail, either return nan's or the bounds...